src/app/modules/shared/components/details-fields/details-fields.component.spec.ts

```diff
@@ -76,6 +76,11 @@ describe('DetailsFieldsComponent', () => {
     description: '',
     technology: '',
   };
+  const dateTest = moment().format('YYYY-MM-DD');
+  const endHourTest = moment().format('HH:mm:ss');
+  const endDateTest = new Date(`${dateTest}T${endHourTest.trim()}`);
+  const startHourTest = moment().subtract(3, 'hours').format('HH:mm:ss');
+  const startDateTest = new Date(`${dateTest}T${startHourTest.trim()}`);
```

Comment:
I would move these two lines before the `end` variables

Reply:
I have change all the tests, please check them.